examples/run_glue_tpu.py

```diff
@@ -177,15 +178,17 @@ def train(args, train_dataset, model, tokenizer, disable_logging=False):
 
                 if args.logging_steps > 0 and global_step % args.logging_steps == 0:
                     # Log metrics.
+                    results = {}
                     if args.evaluate_during_training:
                         results = evaluate(args, model, tokenizer, disable_logging=disable_logging)
-                        for key, value in results.items():
-                            tb_writer.add_scalar('eval_{}'.format(key), value, global_step)
                     loss_scalar = loss.item()
-                    tb_writer.add_scalar('lr', scheduler.get_lr()[0], global_step)
-                    tb_writer.add_scalar('loss', loss_scalar, global_step)
                     logger.info('global_step: {global_step}, lr: {lr:.3f}, loss: {loss:.3f}'.format(
                         global_step=global_step, lr=scheduler.get_lr()[0], loss=loss_scalar))
+                    if xm.is_master_ordinal():
+                        for key, value in results.items():
```

Comment:
similar comment to prev. pr; all the values here need to be on cpu. if so, can you add a comment? it's a subtle point.

Reply:
Done, thanks.